cmd/git-init/main.go

```diff
@@ -39,6 +39,8 @@ func init() {
 	flag.BoolVar(&fetchSpec.Submodules, "submodules", true, "Initialize and fetch Git submodules")
 	flag.UintVar(&fetchSpec.Depth, "depth", 1, "Perform a shallow clone to this depth")
 	flag.StringVar(&terminationMessagePath, "terminationMessagePath", "/tekton/termination", "Location of file containing termination message")
+	flag.StringVar(&fetchSpec.SparseCheckoutDirectories, "sparseCheckoutDirectories", "", "String of directory patterns separated by a delimiter")
+	flag.StringVar(&fetchSpec.SparseCheckoutDelimiter, "sparseCheckoutDelimiter", ";", "Delimiter used to separate sparse checkout directories")
```

Comment:
I don't think we need that level of granularity as a flag

Moreover the delimiter would cause some issues if the user doesn't quote the ";", it will be interepreted as another command.

What about remove the flag and setting the delimiter as a standard comma "," ?

Reply:
That is true, I understand how semicolons could cause issues for users.
Removing the delimiter flag and using a comma will make it simpler.